src/Microsoft.Extensions.CodeGeneration.EntityFrameworkCore/EntityFrameworkServices.cs

```diff
@@ -400,10 +403,10 @@ private DbContext TryCreateContextUsingAppCode(Type dbContextType, ModelType sta
         {
             try {
                 var builder = new WebHostBuilder();
-                //TODO: Review 
-                builder.UseServer("Microsoft.AspNetCore.Server.WebListener")
-                        .UseContentRoot(Directory.GetCurrentDirectory());
-                        
+                builder.UseKestrel()
+                        .UseContentRoot(Directory.GetCurrentDirectory())
+                        .UseIISIntegration();
```

Comment:
This was just to keep it consistent with the Program.cs from templates. I will remove the IIS integration and test if it works.